Formula/rxvt-unicode.rb

```diff
@@ -31,6 +33,11 @@ class RxvtUnicode < Formula
 
   uses_from_macos "perl"
 
+  resource "libptytty" do
+    url "https://github.com/yusiwen/libptytty/archive/b9694ea18e0dbd78213f55233a430325c13ad63e.tar.gz"
+    sha256 "dbe67f6dbe5ba4bb3e0f0a2195b6837aedc32ee6d39d1b5484777a608f8709fb"
+  end
```

Comment:
Why wasn't this needed before?

Reply:
This is a new dependency as of `rxvt-unicode` 9.30.